app/javascript/crayons/MentionAutocompleteTextArea/MentionAutocompleteTextArea.jsx

```diff
@@ -343,6 +343,7 @@ export const MentionAutocompleteTextArea = forwardRef(
         <Combobox
           ref={containerRef}
           id="combobox-container"
+          data-testid="autocomplete-wrapper"
```

Comment:
We were very occasionally getting test flake because the original comment box, and the 'plain  autocomplete comment box' are found int he DOM via the same query (`findByRole('textbox', { name: 'Add to the discussion' })`)

On some rare occasions, the query was finding the original comment box because it hadn't quite been replaced yet. I've used the testid to just make sure we wait for this component to be rendered before continuing with the test run